validator/index.d.ts

```diff
@@ -1,6 +1,6 @@
-// Type definitions for validator.js v5.7.0
+// Type definitions for validator.js v6.2.1
```

Comment:
Weird (@mhegazy?) - anyway, can you just switch this to `v6.2`? After that we can pull this in.